### PR TITLE
vcfdist evaluation

### PR DESCRIPTION
**Pull request by ddsva1**
Created the WDL and docker to run vcfdist on Terra; the read-me was created when adding the WDL to github.
vcfdist is the alternative benchmarker I was comparing to vcfeval, which turned out to be a good tool but not useful to Broad.
Please let me know the feedback you have!!
Thanks,
Dill

### CODE REVIEW

File: BenchmarkVCFs/vcfdistBenchmark.wdl [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+version 1.0
+
+workflow vcfdist_eval {
```
Comment (rickymagner) [inline BenchmarkVCFs/vcfdistBenchmark.wdl:4]: Just a stylistic comment: I'd make workflow/task names using CapitalCamelCase, e.g. `VcfdistEval`, etc. Same for the `.wdl` filename.

**Review by rickymagner — COMMENTED** [review]
This looks great overall! It's nice to have a WDL to run this along with a summary of your analysis done this summer. I just made a couple of mostly stylistic comments/suggestions in a few places. Thanks for doing the analysis for us!

File: BenchmarkVCFs/vcfdistBenchmark.wdl [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+task run_vcfdist_task{
+    input {
+        File truth_vcf
+        File eval_vcf
+        File bed_file
+        File fasta_file
+        
+        String docker
+        Int disk_size_gb = ceil(1 * size(truth_vcf, "GiB") + 10)
```
[the remark this answers is not in the document]
Comment (ddsva1) [reply BenchmarkVCFs/vcfdistBenchmark.wdl]: Nope; have removed it!

File: Utilities/Dockers/Vcfdist/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+RUN ./configure --prefix=/usr/local 
+RUN make
+RUN make install
+RUN export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+#set-up of vcfdist
+RUN git clone https://github.com/timd1/vcfdist
+#RUN cd htslib-1.17/vcfdist/src
+WORKDIR /htslib-1.17/vcfdist/src
+RUN make
+WORKDIR /
```
[the remark this answers is not in the document]
Comment (ddsva1) [reply Utilities/Dockers/Vcfdist/Dockerfile:20]: Added documentation; will edit the 'usage' section once I correct the dockerfile to reflect the rest of the comments?

File: BenchmarkVCFs/vcfdistBenchmark.wdl [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        Int preemptible = 1
+    }
+
+    command <<<
+        /htslib-1.17/vcfdist/src/vcfdist \
+            ~{eval_vcf} \
+            ~{truth_vcf} \
+            ~{fasta_file} \
+            -b ~{bed_file} \
+            -v 0
```
[the remark this answers is not in the document]
Comment (ddsva1) [reply BenchmarkVCFs/vcfdistBenchmark.wdl]: It is a -verbosity flag, the default being 1; I'll set it to an integer input (default value 0, which it's hardcoded to at the moment) and change the vcfdist readme to note the available options (0, 1, 2).

File: BenchmarkVCFs/VcfdistBenchmark.wdl [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    call RunVcfdistTask {
+        input:
+            truth_vcf = truth_vcf,
+            eval_vcf = eval_vcf,
+            bed_file = bed_file,
+            fasta_file = fasta_file,
+            docker = docker,
+            verbosity = select_first([verbosity, 0])
```
Comment (rickymagner) [inline BenchmarkVCFs/VcfdistBenchmark.wdl:20]: I would instead write in the inputs above `Int verbosity = 0` to mean it's optional as an input, but defaults to `0`.

File: Utilities/Dockers/README.md [hunk 6]
```diff
@@ -96,3 +96,14 @@ convenient data manipulation tools.
...
+
+## vcfdist
+
+* Directory: Vcfdist
+* Description: A docker image containing an installation of [vcfdist](https://github.com/TimD1/vcfdist).
+* Location: `us.gcr.io/broad-dsde-methods/vcfdist:v0.1`
+* Used By: [VcfdistBenchmark](../../BenchmarkVCFs/VcfdistBenchmark.wdl)
+* Usage: `vcfdist [COMMAND]`
+* Version Notes:
+  * 1.0: Versions are `htslib-1.17` 1.17, `vcfdist` 2.0.0
```
Comment (rickymagner) [inline Utilities/Dockers/README.md:109]: You can just write `htslib` 1.17 for its version

**Review by rickymagner — APPROVED** [review]
Looks great! Just some very small comments below, but it's looking good, so I will "approve."

File: BenchmarkVCFs/VcfdistBenchmark.wdl (hunk 5 above)
Comment (ddsva1) [reply BenchmarkVCFs/VcfdistBenchmark.wdl]: (in the end I changed the default to 1, as that's what vcfdist uses as the default)

**Pull request merged**